docs/cloud-wallet.md

```diff
@@ -0,0 +1,106 @@
+Here is an improved version of your documentation, with clearer instructions and a more concise description:
+
+---
+
+# Cloud Wallet Documentation
+
+The Cloud Wallet feature allows secure storage and synchronization of wallet documents via an Encrypted Data Vault (EDV).
+
+**File Path:** `@docknetwork/wallet-sdk-core/src/cloud-wallet`
+
+## Feature Overview
+
+The Cloud Wallet integrates with an EDV to securely store, sync, and manage documents. Once initialized, it automatically synchronizes documents between the EDV and the wallet, allowing you to add, update, remove, without dealing with the synchronization logic.
```

Comment:
should we link out to what an EDV is? Or do you think it's safe to assume people know what you're talking about?